docs/blog/_posts/2021-02-17-scala3-rc1.md

```diff
@@ -132,13 +147,14 @@ A lot of work has been done on the metaprogramming side of things. Mostly we are
 - Rename `Not` to `NotGiven` to make its purpose clearer [#10720](https://github.com/lampepfl/dotty/pull/10720)
 - Fix [#10709](https://github.com/lampepfl/dotty/issues/10709): Add missing level check before inlining [#10781](https://github.com/lampepfl/dotty/pull/10781)
 
-# Let us know what you think!
+## Let us know what you think!
+
 If you have questions or any sort of feedback, feel free to send us a message on our
 [Gitter channel](https://gitter.im/lampepfl/dotty). If you encounter a bug, please
 [open an issue on GitHub](https://github.com/lampepfl/dotty/issues/new).
 
+### Contributors
```

Comment:
Why 3rd header level here? In all blog articles, this item is a top-level header.

Reply:
@anatoliykmetyuk Fixed.